scripts/pi-hole/php/header.php

```diff
@@ -249,23 +252,23 @@ function pidofFTL()
             <span class="logo-lg">Pi-<b>hole</b></span>
         </a>
         <!-- Header Navbar: style can be found in header.less -->
-        <nav class="navbar navbar-static-top" role="navigation">
+        <nav class="navbar navbar-static-top">
             <!-- Sidebar toggle button-->
             <a href="#" class="sidebar-toggle" data-toggle="offcanvas" role="button">
                 <span class="sr-only">Toggle navigation</span>
             </a>
             <div class="navbar-custom-menu">
                 <ul class="nav navbar-nav">
-                    <li><a style="pointer-events:none;"><samp><?php echo gethostname(); ?></samp></a></li>
+                    <li><a style="pointer-events:none;"><code><?php echo gethostname(); ?></code></a></li>
```

Comment:
Just been playing about with this, think it would be nice to have the label `hostname: ` here

```suggestion
                    <li><a style="pointer-events:none;">hostname: <code><?php echo gethostname(); ?></code></a></li>
```

Reply:
Not sure about it. It does clear things up but maybe people won't like it because it's more text?

Reply:
I think it is better to have things clearer, even if it means a few more chars.